cmd/options/options.go

```diff
@@ -63,6 +69,7 @@ func (npdo *NodeProblemDetectorOptions) AddFlags(fs *pflag.FlagSet) {
 		[]string{}, "List of paths to system log monitor config files, comma separated.")
 	fs.StringSliceVar(&npdo.CustomPluginMonitorConfigPaths, "custom-plugin-monitors",
 		[]string{}, "List of paths to custom plugin monitor config files, comma separated.")
+	fs.BoolVar(&npdo.EnableK8sExporter, "enable-k8s-exporter", true, "Enables reporting to Kubernetes.")
```

Comment:
What happens when `enable-k8s-exporter` is false? I don't see `enable-k8s-exporter` used anywhere.

Reply:
In `NewK8SExporterOrDie()`, it will return `nil` if this flag is false.